### PR TITLE
 feat(roundtrip): add when and loop logic

### PR DESCRIPTION
**Pull request by liubog2008**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
1. add loop logic
2. add condition logic
```

### CODE REVIEW

**Comment by liubog2008** [comment]
@ddysher

**Comment by ddysher** [comment]
@liubog2008 can u update README.md to introduce the new features

@bbbmj to review the details.

**Comment by bbbmj** [comment]
OK, I will take a detailed review after CIS review.

**Comment by liubog2008** [comment]
@ddysher I will update README after label selector feature #30

File: runtime/roundtrip.go [hunk 1]
```diff
@@ -4,6 +4,18 @@ import (
 	"time"
 )
 
+// DefinitionType defines whether definition is from
```
Comment (bbbmj) [inline runtime/roundtrip.go]: 这句注释好奇怪。。。

Comment (liubog2008) [reply runtime/roundtrip.go]: = = where is from

Comment (liubog2008) [reply runtime/roundtrip.go]: I'll fix it in next PR

**Pull request merged**